services/motion/builtin/builtin_test.go

```diff
@@ -34,6 +36,21 @@ import (
 	viz "go.viam.com/rdk/vision"
 )
 
+func setupMotionServiceFromConfig(t *testing.T, configFilename string) (motion.Service, func()) {
```

Comment:
[nit/opt]: I'd put this in testutils.go

Reply:
We unfortunately cannot; as this function uses `robotimpl`, which imports services/motion/builtin, this function must be in a test file or it causes an import cycle

Reply:
ahh, thank you for explaining :)